include/fluent-bit/flb_custom.h

```diff
@@ -27,6 +27,10 @@
 #include <fluent-bit/flb_metrics.h>
 #endif
 
+/* Custom plugin flag masks */
+#define FLB_CUSTOM_NET_CLIENT   1   /* custom may use upstream net.* properties  */
```

Comment:
wondering if we should keep the same naming style for `upstream` and `downstream` when extending this here, e.g: `FLB_CUSTOM_NET_UPSTREAM` and `FLB_CUSTOM_NET_DOWNSTREAM`, not the best names but to keep the integrity